chartfx-math/src/main/java/de/gsi/math/TRandom1.java

```diff
@@ -17,79 +16,79 @@
+        { 1602690753, 810926582 }, { 885787576, 513818500 }, { 1853512561, 1195205756 }, { 1798585498, 1970460256 },
+        { 1819261032, 1306536501 }, { 1133245275, 37901 }, { 689459799, 1334389069 }, { 1730609912, 1854586207 },
+        { 1556832175, 1228729041 }, { 251375753, 683687209 }, { 2083946182, 1763106152 },
+        { 2142981854, 1365385561 }, { 763711891, 1735754548 }, { 1581256466, 173689858 },
+        { 2121337132, 1247108250 }, { 1004003636, 891894307 }, { 569816524, 358675254 }, { 626626425, 116062841 },
+        { 632086003, 861268491 }, { 1008211580, 779404957 }, { 1134217766, 1766838261 }, { 1423829292, 1706666192 },
+        { 942037869, 1549358884 }, { 1959429535, 480779114 }, { 778311037, 1940360875 }, { 1531372185, 2009078158 },
+        { 241935492, 1050047003 }, { 272453504, 1870883868 }, { 390441332, 1057903098 }, { 1230238834, 1548117688 },
+        { 1242956379, 1217296445 }, { 515648357, 1675011378 }, { 364477932, 355212934 }, { 2096008713, 1570161804 },
+        { 1409752526, 214033983 }, { 1288158292, 1760636178 }, { 407562666, 1265144848 },
+        { 1071056491, 1582316946 }, { 1014143949, 911406955 }, { 203080461, 809380052 }, { 125647866, 1705464126 },
+        { 2015685843, 599230667 }, { 1425476020, 668203729 }, { 1673735652, 567931803 }, { 1714199325, 181737617 },
+        { 1389137652, 678147926 }, { 288547803, 435433694 }, { 200159281, 654399753 }, { 1580828223, 1298308945 },
+        { 1832286107, 169991953 }, { 182557704, 1046541065 }, { 1688025575, 1248944426 },
+        { 1508287706, 1220577001 }, { 36721212, 1377275347 }, { 1968679856, 1675229747 }, { 279109231, 1835333261 },
+        { 1358617667, 1416978076 }, { 740626186, 2103913602 }, { 1882655908, 251341858 }, { 648016670, 1459615287 },
+        { 780255321, 154906988 }, { 857296483, 203375965 }, { 1631676846, 681204578 }, { 1906971307, 1623728832 },
+        { 1541899600, 1168449797 }, { 1267051693, 1020078717 }, { 1998673940, 1298394942 },
+        { 1914117058, 1381290704 }, { 426068513, 1381618498 }, { 139365577, 1598767734 }, { 2129910384, 952266588 },
+        { 661788054, 19661356 }, { 1104640222, 240506063 }, { 356133630, 1676634527 }, { 242242374, 1863206182 },
+        { 957935844, 1490681416 } };
 
     /**
      * default constructor
      */
     public TRandom1() {
         super(0);
         fIntModulus = 0x1000000;
-        fMantissaBit24 = TMath.Power(0.5, 24.);
-        fMantissaBit12 = TMath.Power(0.5, 12.);
+        fMantissaBit24 = Math.pow(0.5, 24.);
+        fMantissaBit12 = Math.pow(0.5, 12.);
         fTheSeeds = fSeed;
         long seed;
         long seedlist[] = { 0, 0 };
 
         fLuxury = 3;
-        int cycle = Math.abs(fgNumEngines / fgMaxIndex);
-        int curIndex = Math.abs(fgNumEngines % fgMaxIndex);
+        int cycle = java.lang.Math.abs(fgNumEngines / fgMaxIndex);
+        int curIndex = java.lang.Math.abs(fgNumEngines % fgMaxIndex);
```

Comment:
Codacy found an issue: [Unnecessary use of fully qualified name 'java.lang.Math.abs' due to existing implicit import 'java.lang.*'](https://app.codacy.com/manual/GSI/chart-fx/pullRequest?prid=5780327)